### PR TITLE
Close changelog for 6.4.3

### PR DESCRIPTION
**Pull request by tsg**
Can be merged right away.

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -29,11 +29,6 @@ https://github.com/elastic/beats/compare/v6.4.1...6.4[Check the HEAD diff]
 
 *Affecting all Beats*
 
-- Fix a race condition with the `add_host_metadata` and the event serialization. {pull}8223[8223] {pull}8653[8653]
-- Fix race condition when publishing monitoring data. {pull}8646[8646]
-- Fix bug in loading dashboards from zip file. {issue}8051[8051]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: Was this a duplicate?

Comment (adriansr) [inline CHANGELOG.asciidoc]: What about this one?

**Comment by ruflin** [comment]
CI failures do not seem to be related.

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (tsg) [reply CHANGELOG.asciidoc]: Yeah, I think we had those already.

Comment (tsg) [reply CHANGELOG.asciidoc]: Thanks, I added it back. Not sure why it fell between the cracks.

**Pull request merged**